### ISSUE TITLE
an error occurred forwarding

### ISSUE THREAD

**Issue by JCzz** [GoogleCloudPlatform/cloud-code-vscode#143]
# Environment:
```
kubectl version
Client Version: version.Info{Major:"1", Minor:"14", GitVersion:"v1.14.7", GitCommit:"8fca2ec50a6133511b771a11559e24191b1aa2b4", GitTreeState:"clean", BuildDate:"2019-09-18T14:47:22Z", GoVersion:"go1.12.9", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{Major:"1", Minor:"14+", GitVersion:"v1.14.6-gke.2", GitCommit:"c9de33b5439df6e206d7ba646787c6ace92d737b", GitTreeState:"clean", BuildDate:"2019-09-06T18:30:33Z", GoVersion:"go1.12.9b4", Compiler:"gc", Platform:"linux/amd64"}
```

Extension version: 0.0.12
VSCode version: 1.38.1
OS: MacOs 10.15
Knative: is install in the same cluster, but I am deploying using the default application.

# Description
I am trying to debug go code with cloud-code, and expected it to work - it actually did once.

# Steps
1. Installed and run this sample from quickstart:
`https://cloud.google.com/code/docs/vscode/quickstart`

2. Deployed Application
3. Started debugger on Kubernetes. It worked the first time, now I get:

`E1004 15:46:53.839334 3003 portforward.go:400] an error occurred forwarding 52194 -> 3000: error forwarding port 3000 to pod b06a5594b2bd8dbf1e80665e5fc2ecda819ebd4a6b4662a0351f72229fc9c1ec, uid : exit status 1: 2019/10/04 13:46:53 socat[1185346] E connect(5, AF=2 127.0.0.1:3000, 16): Connection refused`

**Comment by quoctruong** [comment]
@JCzz Which sample language are you using?

**Comment by JCzz** [comment]
@quoctruong Thanks, I just tried both the go: hello-world and go:Guestbook, same result.

**Comment by quoctruong** [comment]
Ah I think the 2 templates remove the existing debugging entrypoint in anticipation of `skaffold debug` support for Go. Please try adding this line to the `Dockerfile`:
```
ENTRYPOINT ["dlv", "exec", "/app/backend", "--continue", "--accept-multiclient", "--api-version=2", "--headless", "--listen=:3000", "--log"]
```
and comment out the existing `ENTRYPOINT` in the `go: hello world` template and retry.

**Comment by JCzz** [comment]
Thanks @quoctruong 

Thanks man, that was it - I can not believe I did not see the changes in the Dockerfile.

Keep up the good work
/Christian

**Comment by quoctruong** [comment]
@JCzz We reverted the offending changes to the sample application so you don't need to copy from github Dockerfile anymore. You can retry it again with the new sample.

**Comment by JCzz** [comment]
Hi @quoctruong Sounds good with the Dockerfile - thanks! 

FYI, I think I found the reason for the debugger not working the second time - it looks like an issue with the golang debugger, Delve do not releasing the port in the container. Therefore it is not an issue with the Cloud-Code extension. I found a workflow where I just delete the pod(k8s just restart it again fast) and then I can do my debugging again.

Thanks for you reply
